frontend/public/const.js

```diff
@@ -19,3 +19,6 @@ export const EVENTS = {
 export const ALL_NAMESPACES_KEY = '#ALL_NS#';
 export const NAMESPACE_LOCAL_STORAGE_KEY = 'dropdown-storage-namespaces';
 export const LAST_NAMESPACE_NAME_LOCAL_STORAGE_KEY = 'bridge/last-namespace-name';
+
+// Maximal file size user can upload
+export const MAX_FILE_UPLOAD_SIZE = 30;
```

Comment:
I would keep this local to the file input component since

1. It's the only place that it's used.
2. It would be really easy to change this value without changing the error message in the component. Better if they stay together.

I'm confused what "30" is. What is the unit? It should be in the comment at least.

That value seems way to small if this is bytes?

Reply:
It is ... the value should be 4000000B ~ 4MB  .. was my testing value, forgot to put back... sorry for that